kore/app/share/GlobalMain.hs

```diff
@@ -97,9 +104,33 @@ parseKoreProveOptions =
             <> help "The name of the main module in the spec to be proven."
             )
         )
+    <*> parseGraphSearch
     <*> switch
         ( long "bmc"
         <> help "Whether to use the bounded model checker." )
+  where
+    parseGraphSearch =
+        option readGraphSearch
+            (  metavar "GRAPH_SEARCH"
+            <> long "graph-search"
+            <> value BreadthFirst
+            <> help "Search order of the execution graph. Either breadth-first or depth-first. \
```

Comment:
Please use the line length recommended by our coding standards, unless there's an actual reason to go over it: https://github.com/kframework/kore/wiki/Coding-Standards .

You can run `scripts/lint.sh` if you want an automated way to check that you are actually using this line length.

Reply:
done